lib/streamlit/commands/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2018-2020 Streamlit Inc.
```

Comment:
I created a new module for "streamlit commands that don't use DeltaGenerator"; the idea is that the different functions like `st.set_option` and `st.write` should eventually go into their own file here too, instead of bloating the `__init__.py` file.

Reply:
I'm not a fan of the name `commands` but I don't have a better name... My initial thought was api but that's too broad and it could be confusing in terms of a http API. Feel free to ignore